pkg/utils/utils.go

```diff
@@ -48,7 +48,7 @@ func init() {
 	ClusterType = os.Getenv("CLUSTER_TYPE")
 }
 
-func DiscoverSriovDevices() ([]sriovnetworkv1.InterfaceExt, error) {
+func DiscoverSriovDevices(devMode bool) ([]sriovnetworkv1.InterfaceExt, error) {
```

Comment:
at this layer, maybe its worth naming it: "discoverAll" or "withUnsupported" ?

Reply:
changed to "withUnsupported"